### PR TITLE
Generic big hash helper

### PR DESCRIPTION
**Pull request by jfoug**
Handled super large lines in input/.pot files.

!!! Need to drop the 'sample' data before we merge !!!

### CODE REVIEW

**Comment by jfoug** [comment]
!!! Need to drop the 'sample' data before we merge !!!

File: src/loader.c [hunk 1]
```diff
@@ -157,7 +150,7 @@ static void read_file(struct db_main *db, char *name, int flags,
 			}
 		}
 		process_line(db, line);
-		if (ex_size_line)
+		if (ex_size_line!=line_buf)
```
Comment (magnumripper) [inline src/loader.c]: Code style: Please add some air around the operator here.

Comment (magnumripper) [inline src/loader.c]: Same for line 124

File: src/loader.c [hunk 2]
```diff
@@ -1953,7 +1959,12 @@ static void ldr_show_pw_line(struct db_main *db, char *line)
 
 	if (found && show) {
 		if (source[0])
+		{
 			printf("%c%s", db->options->field_sep_char, source);
+			/* with change to fgetll() vs fgets() source may no longer be \n term */
```
Comment (magnumripper) [inline src/loader.c]: Isn't it a latent bug in fgetll() that should be fixed, instead of worked around here?

**Pull request closed without merging**